src/python/tests/core/bot/tasks/corpus_pruning_task_test.py

```diff
@@ -300,7 +300,7 @@ def test_prune(self):
     corpus = os.listdir(self.corpus_dir)
     self.assertEqual(2, len(corpus))
     six.assertCountEqual(self, [
-        '31836aeaab22dc49555a97edb4c753881432e01d',
+        '801c34269f74ed383fc97de33604b8a905adb635',
```

Comment:
This was failing on Python 2 as well, and there is no testcase in the test input which matches this hash.